ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/AuthorizationError.scala

```diff
@@ -14,16 +14,16 @@ object AuthorizationError {
   final case class Expired(authorizedUntil: Instant, currentTime: Instant)
       extends AuthorizationError {
     override val reason =
-      s"Claims were valid until $authorizedUntil, current time is $currentTime."
+      s"Claims were valid until $authorizedUntil, current time is $currentTime"
   }
 
   case object ExpiredOnStream extends AuthorizationError {
-    override val reason = "Claims have expired after the result stream has started."
+    override val reason = "Claims have expired after the result stream has started"
   }
 
   final case class InvalidLedger(authorized: String, actual: String) extends AuthorizationError {
     override val reason =
-      s"Claims are only valid for ledgerId '$authorized', actual ledgerId is '$actual'."
+      s"Claims are only valid for ledgerId '$authorized', actual ledgerId is '$actual'"
   }
 
   final case class InvalidParticipant(authorized: String, actual: String)
```

Comment:
Note: the change here is not done here, but in https://github.com/digital-asset/daml/pull/12343 to avoid a merge conflict.